### ISSUE TITLE
MD033 triggered on Automatic Links for email addresses

### ISSUE THREAD

**Issue by bvhme** [markdownlint/markdownlint#263]
Hey, In my project we use automatic links for email addresses, e.g. `<example@example.net>` because it reads most easily in the raw markdown (and that's why we use markdown). However, this triggers MD033 as a detected inline HTML. Same for domains without a protocol.

My markdown renderers pick up on this fine but the linter trips. (the Node one doesn't).

**Comment by DavidAnson** [comment]
In case it helps, this syntax is almost as clear and maybe won’t trip the Ruby implementation (the Node implementation is okay with it): `<example@example.net>`.

**Comment by bvhme** [comment]
Yeah, I see the Ruby implementation isn't, so perhaps this is a natural point to switch.

**Comment by jaymzh** [comment]
@DavidAnson  how are you exempting that from your inline HTML rule?

**Comment by DavidAnson** [comment]
Lines 12 and 30: https://github.com/DavidAnson/markdownlint/blob/master/lib/md033.js

**Comment by jaymzh** [comment]
Hmmm, @bvhme I can't repro. Given this test file:

```
# some title

This is a line.
  
This is a line with an <email@address.com> that exists on a line.

Bla bla.
```
I get no errors:

```
[phil@rider (master) markdownlint]$ bin/mdl /tmp/test.md 
[phil@rider (master) markdownlint]$ 
```

Do you have an example that trips it?

**Comment by bvhme** [comment]
I just tried to reproduce and I don't get the same result as all, if this was user error I apologise, if not I'll expland on this report in the future.

My apologies.
